xds/core/v3/authority.proto

```diff
@@ -1,18 +1,18 @@
 syntax = "proto3";
 
-package udpa.core.v1;
+package xds.core.v3;
 
 import "udpa/annotations/status.proto";
```

Comment:
It looks like this PR is just changing the URI-scheme-related protos, but leaving things like the annotation protos in the old paths.  Is there any reason for that?  It seems like it would be easier for clients to just migrate everything at once.

Reply:
The URI scheme is not really in use anywhere yet. The annotations are in extensive use in the Envoy repository. We need to do some major work, including tooling and probably compatibility investigation to move these ones.

Reply:
What kind of tooling and compatibility investigation is needed?  It's not clear to me why this isn't as simple as a global search-and-replace and then bumping the commit that Envoy is importing.

Reply:
I guess the question is will we break anyone who is using these annotations in tooling outside of Envoy? Does this violate our backwards compatibility story for API version stability? That's the main blocking issue IMHO.

Reply:
I assume that anyone who's importing these protos is pinning their import to a particular commit (e.g., via a git submodule or bazel dependency).  If so, changing this now is not going to break existing users; it just means that they'll need to update their code to use the new names as part of updating to a later commit.

Reply:
Right, but isn't this a similar argument to that made around not breaking language bindings in the API?

Reply:
Yeah, good point.  But then can we ever change to use the new protos?  Or will have to to do a field-by-field deprecation-and-replacement?

Reply:
I think we can put new annotations in the `xds.annotations` namespace. For the existing ones, it's not a huge deal to keep as is, but we could clean up if we have consensus (CC @mattklein123 @lizan) on whether we can do breaking changes for annotations. It's a case not covered in https://github.com/envoyproxy/envoy/blob/master/api/API_VERSIONING.md#backwards-compatibility, my comment above appealed to the `oneof` case as precedent, but I think it's fairly likely that tooling outside of Envoy repository doesn't care about these annotations.

Reply:
This one can't change easily; https://github.com/cncf/udpa/blob/master/udpa/type/v1/typed_struct.proto this is not an annotation.

This https://github.com/cncf/udpa/blob/master/udpa/annotations/sensitive.proto is probably the only annotation I expect others to potentially be consuming.